gensim/utils.py

```diff
@@ -1037,10 +1023,10 @@ def lemmatize(content, allowed_tags=re.compile('(NN|VB|JJ|RB)'), light=False,
     ['rank/NN', 'study/VB', 'hard/RB']
 
     """
-    if not has_pattern():
-        raise ImportError("Pattern library is not installed. Pattern library is needed in order  \
-         to use lemmatize function")
-    from pattern.en import parse
+    try:
+        from pattern.en import parse
+    except ImportError:
+        warnings.warn("Pattern library is not installed, lemmatization won't be available.")
```

Comment:
yeah, it will create an unhandled expection later in the method where parse is used, since only a warning is created. I'll change that.